package.json

```diff
@@ -74,6 +77,8 @@
     "react-lazyload": "^3.1.0",
     "react-map-gl": "^5.2.8",
     "react-map-gl-draw": "^0.21.1",
```

Comment:
Also could not find usage of `react-map-gl-draw` anymore, right?

Reply:
Yes removing that. we don't need anymore. `react-mapbox-gl-draw` replaces it.